spec/lcm/integration/user_provisioning_end_to_end_spec.rb

```diff
@@ -0,0 +1,172 @@
+require_relative 'support/constants'
+require_relative 'support/configuration_helper'
+require_relative 'support/connection_helper'
+require_relative 'support/lcm_helper'
+require_relative 'support/project_helper'
+require_relative 'support/s3_helper'
+require_relative 'brick_runner'
+
+def uniq_label_with_value(project)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)